CompetitionTemplate/src/competition/subsystems/collector/commands/CollectorIntakeCommand.java

```diff
@@ -29,7 +32,7 @@ public void end() {
     @Override
     public void execute() {
         if (enableCollectorAutoStop.get()) {
-            if (!collectorSubsystem.isBallInCollector()) {
+            if (this.timeSinceInitialized() < autoStopOverrideTime.get() || !collectorSubsystem.isBallInCollector()) {
```

Comment:
this seems reasonable, but I wonder if the behavior being corrected indicates the ball sensor string needs tuning physically?